### PR TITLE
closes #39 added working carousel button

### LINKED ISSUE #39: Clicking the category button on the carousel should work
# AC
Clicking the category button on the carousel should link to `products/category/${that_category}`

### PR DESCRIPTION
**Pull request by MissQuinn-dev**

### CODE REVIEW

File: src/components/products/ProductCategory.jsx [hunk 1]
```diff
@@ -16,6 +16,8 @@ const ColorButton = styled(Button)(({ theme }) => ({
 }));
 
 const ProductCategory = ({ category, image }) => {
+  console.log(category);
```
Comment (justJay-dev) [inline src/components/products/ProductCategory.jsx:19]: remove console.log here

**Review by justJay-dev — CHANGES_REQUESTED** [review]
remove a console.log and you are good!

File: src/components/products/ProductCategory.jsx (hunk 1 above)
Comment (MissQuinn-dev) [reply src/components/products/ProductCategory.jsx]: removed!

**Review by justJay-dev — APPROVED** [review]
LGTM!

**Pull request merged**